### PR TITLE
Set Up couchDB connection, able to seed with 100 items

### PR DESCRIPTION
**Pull request by masoncpott**
@sam-masno @sjamcox @tyler-thee-creator 

-set up a connection to couchDB
-wrote a seeding function and script that creates the connection, then creates fake data, then puts the fake data into the database
-i have confirmed this works by checking the couchDB Fauxton for the records and performing queries on them.

### CODE REVIEW

File: checkout-service/database/couchConnection.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const { username, password, port } = require('./config.js');
+const nano = require('nano')(`http://${username}:${password}@localhost:${port}`);
+
+const createAndConnectToDatabase = async () => {
+  try {
```
Comment (tyler-thee-creator) [inline checkout-service/database/couchConnection.js:5]: Nice job w/ the catch block that catches all errors from all steps in creating DB, also like how this is its own function

File: checkout-service/database/couchConnection.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    // succeeded
+    console.log('Successfully created and using database!');
+    return database;
+  } catch (e) {
+    // failed
+    console.error('Error creating database!', e);
+  }
+};
+
+module.exports = { createAndConnectToDatabase };
```
Comment (tyler-thee-creator) [inline checkout-service/database/couchConnection.js:18]: Never seen destructuring used w/ the module.exports but this is clean, I'm gonna start doing this

File: checkout-service/database/seed/couchPopulate.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const CreateFakeData = require('./dataGenerator.js');
+const { createAndConnectToDatabase } = require ('../couchConnection.js');
+const nano = require('nano');
+
+const DbSeed = async function() {
```
Comment (tyler-thee-creator) [inline checkout-service/database/seed/couchPopulate.js:5]: My seed function was nearly the same in its approach, nicely done!

**Review by tyler-thee-creator — APPROVED** [review]
Looked over it all and looks solid

File: checkout-service/database/couchConnection.js (hunk 2 above)
Comment (masoncpott) [reply checkout-service/database/couchConnection.js:18]: if the key and value in an object are exactly the same, my understanding is that you only have to list one of them, for example: 
this { exampleFunction: exampleFunction } is the same as { exampleFunction }

**Pull request merged**